command/fileserver/fileserver.go

```diff
@@ -121,6 +122,7 @@ func fileServerAction(ctx *cli.Context) error {
 		tlsConfig = &tls.Config{
 			ClientCAs:  pool,
 			ClientAuth: tls.RequireAndVerifyClientCert,
+			MinVersion: tls.VersionTLS13,
```

Comment:
I'm ok with requiring 1.3, but as this is mainly used for testing 1.2 might be more useful.